src/HotChocolate/Language/src/Language.SyntaxTree/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -1,7 +1,6 @@
-using System;
 using System.Runtime.CompilerServices;
 
-namespace HotChocolate.Language;
+namespace HotChocolate.Language.Extensions;
```

Comment:
This is wrong ... please do not change the namespaces.

Reply:
ops, my fault. Thought it wasn't migrated. Returning it back